source/code-snippets/usage-examples/findOne.js

```diff
@@ -13,7 +13,7 @@ async function run() {
       const database = client.db('sample_mflix');
       const collection = database.collection('movies');
 
-      // create a query document to look up an exact match for a move with this title
+      // create a query document to look up any exact match for a movie with this title
```

Comment:
"exact match" is a bit confusing because I could interpret that to mean that the document is literally identical to the query (no additional fields, etc). Maybe:

`// Query for a movie that has the title "The Room"`